src/commands/pack/win.ts

```diff
@@ -228,47 +236,6 @@ the CLI should already exist in a directory named after the CLI that is the root
     }),
   }
 
-  async run(): Promise<void> {
-    await this.checkForNSIS()
-    const {flags} = await this.parse(PackWin)
-    const buildConfig = await Tarballs.buildConfig(flags.root, {targets: flags?.targets?.split(',')})
-    const {config} = buildConfig
-    await Tarballs.build(buildConfig, {platform: 'win32', pack: false, tarball: flags.tarball, parallel: true})
-    const arches = buildConfig.targets.filter(t => t.platform === 'win32').map(t => t.arch)
-    const nsisCustomization = config.nsisCustomization ? fs.readFileSync(config.nsisCustomization, 'utf8') : ''
-
-    await Promise.all(arches.map(async arch => {
-      const installerBase = path.join(buildConfig.tmp, `windows-${arch}-installer`)
-      await fs.promises.rm(installerBase, {recursive: true, force: true})
-      await fs.promises.mkdir(path.join(installerBase, 'bin'), {recursive: true})
-      await Promise.all([
-        fs.writeFile(path.join(installerBase, 'bin', `${config.bin}.cmd`), scripts.cmd(config)),
-        fs.writeFile(path.join(installerBase, 'bin', `${config.bin}`), scripts.sh(config)),
-        fs.writeFile(path.join(installerBase, `${config.bin}.nsi`), scripts.nsis(config, arch, nsisCustomization)),
-      ].concat(config.binAliases ? config.binAliases.flatMap(alias =>
-        // write duplicate files for windows aliases
-        // this avoids mklink which can require admin privileges which not everyone has
-        [fs.writeFile(path.join(installerBase, 'bin', `${alias}.cmd`), scripts.cmd(config)), fs.writeFile(path.join(installerBase, 'bin', `${alias}`), scripts.sh(config))]) : [])
-      .concat(flags['additional-cli'] ? [
-        fs.writeFile(path.join(installerBase, 'bin', `${flags['additional-cli']}.cmd`), scripts.cmd(config, flags['additional-cli'])),
-        fs.writeFile(path.join(installerBase, 'bin', `${flags['additional-cli']}`), scripts.sh({bin: flags['additional-cli']} as Interfaces.Config)),
-      ] : []))
-
-      await fs.move(buildConfig.workspace({platform: 'win32', arch}), path.join(installerBase, 'client'))
-      await exec(`makensis ${installerBase}/${config.bin}.nsi | grep -v "\\[compress\\]" | grep -v "^File: Descending to"`)
-      const templateKey = templateShortKey('win32', {bin: config.bin, version: config.version, sha: buildConfig.gitSha, arch})
-      const o = buildConfig.dist(`win32/${templateKey}`)
-      await fs.move(path.join(installerBase, 'installer.exe'), o)
-
-      const windows = (config.pjson.oclif as any).windows as {name: string; keypath: string; homepage?: string}
-      if (windows && windows.name && windows.keypath) {
-        await signWindows(o, arch, config, windows)
-      } else this.debug('Skipping windows exe signing')
-
-      this.log(`built ${o}`)
-    }))
-  }
-
   private async checkForNSIS() {
```

Comment:
I guess it's generally alphabetizing object properties?  It's weird not having `run` as the first method on a command.

And putting that private checkForNSIS first is...weird.